tidy3d/web/simulation_task.py

```diff
@@ -204,10 +212,12 @@ def create(
 
         Parameters
         ----------
-        simulation: :class".Simulation"
+        stub: :class".Stub"
```

Comment:
so we will still use this SimulationTask and other things in transition? eventually, it would be easiest to have a single `StubTask` and just use that for any Stub subclasses (eg mode solver) is that right?

Reply:
SimulationTask is your StubTask by now.